lib/index.js

```diff
@@ -54,20 +65,23 @@ class PWMetrics {
           results[index] = data;
         }).catch(error => {
           console.error(getMessageWithPrefix('ERROR', 'FAILED_RUN', index, results.length, error.message));
-          throw new Error(error);
+          console.error(error.stack + '\n');
+          results[index] = error;
         });
       });
     }, Promise.resolve());
 
     return allDone.then(_ => {
-      const ret = {runs: results};
-      if (this.runs > 1 && !this.flags.submit) {
-        ret.median = this.findMedianRun(results);
+      const ret = {runs: results.filter(r => !(r instanceof Error))};
+      if (this.runs > 1 && !this.flags.submit && ret.runs.length) {
```

Comment:
We already have it, I merely don't include the failed runs so it doesn't crash the findMedian function